src/components/Login/Login.tsx

```diff
@@ -85,21 +94,29 @@ function Login() {
             return res.json();
           } else {
             return res.json().then(() => {
-              const errorMessage = 'Authentication failed';
-              throw new Error(errorMessage);
+              notification.open({
+                message: 'Authentication Failed',
+                description: 'Wrong Username or Wrong Password',
+              });
             });
           }
         })
         .then((data) => {
           console.log(data);
           console.log(data.idToken);
-          alert('Welcome ' + data.email);
+          notification.open({
+            message: 'Welcome ' + data.email,
+            description: 'Successfully Signed in',
+          });
           authCtx.login(data.idToken);
           console.log(authCtx);
           history('/films');
         })
         .catch((err) => {
-          alert(err.message);
+          notification.open({
```

Comment:
error ? and why do you have these kinds of checks for username and pass ? what is the purpose ?